src/game/shared/makeAnimation.ts

```diff
@@ -0,0 +1,17 @@
+function makeAnimation(
+  target: (frame: number) => void,
+  frames: number[],
+  frameInterval: number
+): void {
+  let step = 0;
+  const srt = setInterval(() => {
```

Comment:
❗️
Лучше отбить пустой строкой
```suggestion
  let step = 0;
  
  const srt = setInterval(() => {
```